### DISCUSSION TITLE
[Implemented] Origin Reminder

### DISCUSSION THREAD

**Discussion by HDXylophone** [cheeeeeeeeeen/RoR2-ClassicItems#27] [Ideas]
Reminder about the issue mentioned in the discord, about Origin seemingly spawning additional overlords for each player.

**Comment by cheeeeeeeeeen** [comment]
https://discord.com/channels/562704639141740588/567832879879553037/786434709060190230

Related discord message by rob.

**Comment by cheeeeeeeeeen** [comment]
This feature will be implemented. See #29.

Open a new discussion if there needs to be some changes.
